cmd/boulder-va/main.go

```diff
@@ -119,18 +123,30 @@ func main() {
 		resolver = r
 	}
 
+	tls, err := c.VA.TLS.Load()
+	cmd.FailOnError(err, "TLS config")
+
+	var remotes []va.RemoteVA
+	if len(c.VA.RemoteVAs) > 0 {
+		for _, rva := range c.VA.RemoteVAs {
+			vaConn, err := bgrpc.ClientSetup(&rva, tls, scope)
+			cmd.FailOnError(err, "Unable to create remote VA client")
+			remotes = append(remotes, va.RemoteVA{bgrpc.NewValidationAuthorityGRPCClient(vaConn), strings.Join(rva.ServerAddresses, ",")})
```

Comment:
Alright, sounds good. But please add a comment to RemoteVA explaining what it's for.